src/lib/components/PrismicText.svelte

```diff
@@ -1,12 +1,18 @@
 <script lang="ts">
   import * as prismicH from "@prismicio/helpers"
+  import type * as prismicT from "@prismicio/types";
 
-  import type { RichTextField as RichTextFieldType } from "@prismicio/types";
-  import type { TitleField as TitleFieldType } from "@prismicio/types";
+  /**
+	 * A Prismic Rich Text or Title field.
+	 */
+  export let field:prismicT.TitleField | prismicT.RichTextField
 
-  export let field:TitleFieldType | RichTextFieldType
-  export let separator:string
+  /**
+	 * The separator used between blocks. Defaults to `\n`.
+   * Newlines will not render unless `white-space: pre-line` is 
+   * applied with CSS.
+	 */
+  export let separator:string | undefined = `\n`;
```

Comment:
Style: <code>\`</code> should only be used for string interpolation (e.g. <code>\`foo${bar}\`</code>). In this case `"` should be used instead:

```ts
export let separator: string = "\n";
```